contacts.py

```diff
@@ -382,17 +425,19 @@ def _insert_blob_with_optional_replacement(self, table, blob, floating_seconds):
     @register_method(route='/status/send')
     def send_status(self, data, args):
         logger.info('in send_status')
-        floating_seconds = current_time()
+        # floating_seconds = current_time() # use separate time for each point
 
         # These are fields allowed in the send_status, and just copied from top level into each data point
         # Note memo is not supported yet and is a placeholder
         # first process contacts, then process geocode
         repeated_fields = {k: data.get(k) for k in ['memo', 'replaces', 'status'] if data.get(k)}
         for contact in data.get('contacts_id', []):
             contact.update(repeated_fields)
+            floating_seconds = current_time()
```

Comment:
we need to be careful that if we move to the twisted clock (which doesn't do an os call each time, but updates it as part of the reactor).. to discuss.